src/HomeWork/HomeWork5/Part2/Pet.java

```diff
@@ -0,0 +1,27 @@
+package HomeWork.HomeWork5.Part2;
+
+public class Pet extends Animal{
+    String name;
+    int tail = 1;
+    int paw = 4;
+
+    public Pet(String name) {
```

Comment:
A constructor is needed for a new object creation. It's no needed to call some methods here